### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Innocent259**
In this project milestone I:

1. Applied styling using Bootstrap to meet the given design

### CODE REVIEW

File: src/Components/NewBook.js [hunk 1]
```diff
@@ -30,23 +30,34 @@ export default function NewBook() {
...
+              name="author"
+              value={bookData.author}
+              placeholder="Author"
+              onChange={handleInputChange}
+            />
+          </div>
+          <div className="col-md-2">
+            <button className="btn btn-primary btn-sm w-100" type="submit">ADD BOOK</button>
+          </div>
+        </div>
```
Comment (Ibizugbe) [inline src/Components/NewBook.js]: Kindly style your form section to match the given design as it is a requirement in this milestone that our project matches the given design😉.

yours
image

design
image

Kindly take note of the font properties, borders, width and height of each elements.

File: src/Components/BookDetails.js [hunk 2]
```diff
@@ -13,29 +15,42 @@ export default function BookDetails({ id, title, author }) {
...
+            <div className="inner-b" />
+          </div>
+          <div className="d-flex flex-column">
+            <p className="mb-0 fs-5">64%</p>
+            <small className="fs-6 text-muted">completed</small>
+          </div>
+        </div>
+        <div className="col-md-3">
+          <div>
+            <h3 className="fs-6 text-muted">Current Chapter</h3>
```
Comment (Ibizugbe) [inline src/Components/BookDetails.js]: Let's style each card to match the design as this is a requirement for this milestone😉😉.

design
image

yours
image

please take note of the font properties, spacings and ensure it matches the design😉.

File: src/Components/Navbar.js [hunk 3]
```diff
@@ -15,16 +16,16 @@ export default function Navbar() {
...
-                <Link className="nav-link" to="/categories">Categories</Link>
+                <Link className="nav-link fs-5" to="/categories">Categories</Link>
               </li>
             </ul>
           </div>
         </div>
-        <div className="d-none d-md-block">
-          right-img
+        <div className="d-none d-md-block border border-secondary rounded-circle profile">
+          <div className="text-primary text-center fs-3"><BsFillPersonFill /></div>
```
Comment (Ibizugbe) [inline src/Components/Navbar.js]: Nice job styling your project, but let's put in more effort to ensure the Nav section matches the design as this is a requirement for this milestone😉.

yours
image

design
image

Take note of the font properties and the spacings and make the changes where necessary😉😉.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Innocent259 👋🏼👋🏼👋🏼👋🏼, 

require_changes


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅✅

- [x] Linters are passing👍🏼.
- [x] Nice effort making your project match the given design👍🏼.
- [x] Gitflow is used👍🏼.
- [x] Commit messages are descriptive👍🏼.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @Innocent259 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewer
- Good design.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**